src/hearings/containers/request-hearing/hearing-timing/hearing-timing.component.ts

```diff
@@ -66,17 +68,47 @@ export class HearingTimingComponent extends RequestHearingPageFlow implements On
 
   public ngOnInit(): void {
     this.initDateConfig();
+    this.getFormData();
     this.initForm();
-    this.priorities = this.route.snapshot.data.hearingPriorities.sort((currentPriority, nextPriority) => (currentPriority.order < nextPriority.order ? -1 : 1));
+    this.priorities = this.route.snapshot.data.hearingPriorities.sort((currentPriority: { order: number; }, nextPriority: { order: number; }) => (currentPriority.order < nextPriority.order ? -1 : 1));
     this.hearingStore.pipe(select(fromHearingStore.getHearingValuesModel),
-      map((hearingValues): PartyUnavailabilityRange[] => hearingValues && hearingValues.parties && hearingValues.parties.map(dates => dates.unavailability).flat()))
+      map((hearingValues): PartyUnavailabilityRange[] => hearingValues && hearingValues.parties && hearingValues.parties.map((dates: { unavailability: any; }) => dates.unavailability).flat()))
       .subscribe((unavailabilityDateList: PartyUnavailabilityRange[]) => {
         if (unavailabilityDateList) {
           this.checkUnavailableDatesList(unavailabilityDateList);
         }
       });
```

Comment:
We can call this.serviceHearingValuesModel.privateHearingRequiredFlag directly now as the subscription block is done in parent component.